streams/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java

```diff
@@ -155,4 +163,20 @@ public void createTopic(final String topic,
     public void deleteTopic(final String topic) {
         brokers[0].deleteTopic(topic);
     }
+
+    public void waitForTopics(final StreamsConfig config, final String...topics) throws InterruptedException, IOException {
+        final StreamsKafkaClient streamsKafkaClient = new StreamsKafkaClient(config);
```

Comment:
Ok - i've moved it to `IntegrationTestUtils` as the `org.apache.kafka.streams.integration` package is already allowed to import most of these. I've had to add 1 more package to the allowed imports.